src/Databases/DatabaseReplicated.cpp

```diff
@@ -173,13 +173,40 @@ ClusterPtr DatabaseReplicated::tryGetCluster() const
     return cluster;
 }
 
-void DatabaseReplicated::setCluster(ClusterPtr && new_cluster)
+ClusterPtr DatabaseReplicated::tryGetAllGroupsCluster() const
 {
     std::lock_guard lock{mutex};
-    cluster = std::move(new_cluster);
+    if (replica_group_name.empty())
+        return nullptr;
```

Comment:
should it just return regular getClusterImpl when we accidentally call tryGetAllGroups on a clickhouse instance without replicaGroup?